servicebroker/profiles/hcf.properties

```diff
@@ -0,0 +1,23 @@
+service.name=CF-AutoScaler
+
+#xserverURIList=AutoScaling.bosh-lite.com
+#xapiServerURI=AutoScalingAPI.bosh-lite.com
+serverURIList=autoscaler-server.<%= p('system_domain') %>:28862/server
+apiServerURI=autoscaler-api.<%= p('system_domain') %>:28861/api
```

Comment:
These shouldn't need to be exposed via the router; something like this should work:
```
+serverURIList=autoscaler-server:28862/server
+apiServerURI=autoscaler-api:28861/api
```
(But they'd need to be configurable, because in BOSH-land they'd be `autoscaler-server.services.cf.internal`)